### ISSUE TITLE
Enable markup delegation of client hints

### ISSUE THREAD

**Issue by arichiv** [WICG/client-hints-infrastructure#73]
We should allow permission policies to be set on Accept-CH meta tags to enable third party contexts to receive client hints. For example:
`<meta name="accept-ch" content="sec-ch-ua-platform-version=('self' 'https://foo.bar')">`

This would support cases in which third party content is adaptable to the client and such adaptations improve user experience. [Variable fonts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide), [color vector fonts](https://www.chromestatus.com/feature/5638148514119680), and [responsive images](https://github.com/w3c/webappsec-permissions-policy/issues/55#issuecomment-406627096) could all be enabled by this change.

Here's an early design doc: https://docs.google.com/document/d/1U3P9yvaT1NXG_qRmY3Lp6Me7M5kTnd3QrBb1yFUVNNk/edit

**Comment by arichiv** [comment]
@miketaylr, @clelland, @yoavweiss, & @eeeps PR will be up tomorrow

**Comment by eeeps** [comment]
It's been very exciting to see Chrome's progress implementing this, here: https://bugs.chromium.org/p/chromium/issues/detail?id=1219359

Any guesses as to when this WIP feature might be enabled by default? Cloudinary is planning on shipping features that build on responsive image client hints in Q1. We’re trying to be really conscientious about what we do when we don’t receive hints... but the more hints we receive, the better, and this feature looks like it will make it much easier for many more people to enable them.

**Comment by arichiv** [comment]
TL;DR The goal is M99

We're almost code complete (subresources can get client hints delegated in HTML in canary, and iframes should get them next week). That said, intents and launches take longer around the holidays so there's a chance it'll stay experimental only (chrome://flags/#enable-experimental-web-platform-features).
